core/src/main/java/org/apache/metamodel/util/ResourceUtils.java

```diff
@@ -18,11 +18,31 @@
  */
 package org.apache.metamodel.util;
 
+import java.net.URI;
+
+import org.apache.metamodel.factory.ResourceFactoryRegistryImpl;
+import org.apache.metamodel.factory.ResourceProperties;
+import org.apache.metamodel.factory.SimpleResourceProperties;
+import org.apache.metamodel.factory.UnsupportedResourcePropertiesException;
+
 /**
  * Static utility methods for handling {@link Resource}s.
  */
 public class ResourceUtils {
 
+    public static Resource toResource(URI uri) {
+        return toResource(new SimpleResourceProperties(uri));
+    }
+
+    public static Resource toResource(String uri) {
+        return toResource(new SimpleResourceProperties(uri));
+    }
+
+    public static Resource toResource(ResourceProperties resourceProperties)
+            throws UnsupportedResourcePropertiesException {
```

Comment:
Ok so there are arguments on both sides here. But then I can throw another one: For consistency it makes sense to use the 'throws' keyword since that's what is being done in other core interfaces of MetaModel, including the most core of all: `DataContext` (has many `throws MetaModelException` declarations).